proposals/lip-0044.md

```diff
@@ -637,8 +637,8 @@ The Validators module does not execute any logic during the genesis block proces
 This LIP defines a new module and specify its store, which in turn will become part of the state tree and will be authenticated by the state root. As such, it will induce a hardfork.
 
 ## Reference Implementation
-
-TBA
+- [Introduce Validators module](https://github.com/LiskHQ/lisk-sdk/issues/6668) 
+- https://github.com/LiskHQ/lisk-sdk/tree/development/framework/src/modules/validators
```

Comment:
@sitetester Linking the whole folder or a file will not give a clear picture of the implementation as there could be a lot of implementation that might be irrelevant to LIP and also code maybe evolved over time due to subsequent LIPs in future. We can can use tag instead of branch. 
- Add Epic link corresponding to a LIP; epic has all the issues to implement a LIP and related PRs can be easily navigated through the issues.
- If the Epic is not present then there should be a related issue(s)/PR(s) that can be linked
- We can remove the author name as those names can be checked on PRs

Regarding difficulty for the reviewer to review each PR, I think in any case we should review it to make sure all the features described in the LIP were implemented and we can always navigate easliy to relevant LIP implementation. Reviewer only need to check whether the PR is relevant or not instead of code as its already reviewed thoroughly.